rocketpy/motors/Motor.py

```diff
@@ -5,7 +5,9 @@
 __license__ = "MIT"
 
 import re
+import warnings
 from abc import ABC, abstractmethod
+from rocketpy.utilities import tuple_handler
```

Comment:
This behavior is not accepted anymore. 

The `tuple_handler` function should be positioned in the tools.py file, otherwise an importError would be raised in the future due to circular reference.

Reply:
Solved by [1a2959e](https://github.com/RocketPy-Team/RocketPy/pull/337/commits/1a2959e0f1695f13ffa9a47b4e28864e8c0b5a17)